infer/src/IR/Procdesc.mli

```diff
@@ -38,6 +38,8 @@ module Node : sig
   (** kind of statement node *)
   type stmt_nodekind =
     | AssertionFailure
+    | AtomicCompareExchangeBranch
+    | AtomicExpr of string
```

Comment:
A variant would be better than a string here. Or you could just leave out the argument to avoid copying the large type over to this file. This type is only used for debug anyway. The reason that a string is not ideal is that these end up in the CFGs written to disk and a string takes a lot more space than a variant (which is just an int).

Reply:
I just removed the argument, which let me delete the massive `atomic_expr_to_string` function too.